manuscript/manuscript.Rmd

```diff
@@ -448,6 +449,7 @@ Methods which must first assume something about the structure or functional form
 Frequently available ecological data is insufficient to distinguish between possible alternative models [@knape_are_2012], or the correct model may be non-identifiable with any amount of data.
 Model-free RL approaches offer a powerful solution for this thorny issue.
 Model-free algorithms have proven successful at learning effective policies even when the underlying model is difficult or impossible to learn [@Pong2020], as long as simulations of possible mechanisms are available.
+Yet, model-free algorithms tend to have poor sample efficiency [@sac] which may restrain them from being insightful on simulators that are slow to run. 
```

Comment:
Good to get this in here and probably fine as it is.  Just a few quick thoughts: (1) would this be better to bring up in the discussion (we have plenty of word-space now).  (2) should this be "current model-free algorithms" or something like that?

This is definitely an important issue to highlight, particularly if we face another reviewer who expects to see arbitrarily complex examples shown here.  At the same time, it feels a little premature to have a firm verdict about this (maybe techniques to improve sample efficiency will develop; or maybe computational power will continue to enable approaches that have previously been deemed to inefficient).  

Kinda highlights the general challenge we have of walking the line between what is practical today and what may be within reach in the next decade...

Reply:
Yup valid point. Another thing I've been trying to work in is how to convey that RL could learn on sampled data via offline RL but open question on how much is enough. I guess this might be an aside that I could make in the empirical data section. This general point of when someone can use RL or not came up as a slight point of confusion with the control group.

Reply:
yeah, the offline idea is really interesting, definitely seems possible.  (I think particularly in problems with big state spaces but small action spaces; which is how a lot conservation / global change problems are currently framed at least.)  

Maybe we want a discussion paragraph that tackles the "where can I use RL today" more head-on.  I guess the trouble is that which problems "can use" deep RL probably depends pretty heavily on what resources you have available.  I'd like to convey that (a) small teams can get started by just turning simple existing ecological models into gyms and trying existing algorithms, like we do; but also (b) applying deep RL to real-world problems is going to require a major investment of time, resources, and probably large interdisciplinary teams; this probably isn't kind of thing a single PhD student can do on laptop.